gitlab/resource_gitlab_user.go

```diff
@@ -182,6 +188,22 @@ func resourceGitlabUserUpdate(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
+	if d.HasChange("state") {
+		if d.Get("state") == "active" {
+			err := client.Users.UnblockUser(id)
+
+			if err != nil {
+				return err
+			}
+		} else if d.Get("state") == "deactivated" {
```

Comment:
This should be `blocked` not `deactivated` right?

Reply:
Thanks for catching that, fixed.